openbook_posts/models.py

```diff
@@ -636,6 +660,39 @@ def create_post_comment_mute(cls, post_comment_id, muter_id):
         return cls.objects.create(post_comment_id=post_comment_id, muter_id=muter_id)
 
 
+class PostLink(models.Model):
+    uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
```

Comment:
Why use an uuid? 🤔

Reply:
I had a weird error from the mysql database about the key length.  I think since i have a unique constraint on the post and link, it tries to use that as the primary key.. but since the link field has max length = post max length, the primary key max value is too long for the database.

So adding a uuid gave it another primary key.